ingestion/src/metadata/ingestion/source/glue.py

```diff
@@ -171,6 +171,15 @@ def ingest_tables(self, next_tables_token=None) -> Iterable[OMetaDatabaseAndTabl
                         id=self.storage_service.id, type="storageService"
                     ),
                 )
+
+                table_type: TableType = None
+                if table["TableType"] == "EXTERNAL_TABLE":
+                    table_type = TableType.External
+                elif table["TableType"] == "VIRTUAL_VIEW":
+                    table_type = TableType.View
+                else:
+                    table_type = TableType.Regular
```

Comment:
Hi, @ulixius9 thanks a lot for this PR!!

Instead of initializing the `table_type` as `None` and filling it up in the different conditions, do you think that we could start with the `TableType.Regular` as the default and then check for external or view?

Might be a bit easier to read.

Thanks 🙏